files/identity/launch-pelion-identity.sh

```diff
@@ -28,7 +27,7 @@ false
 while [ $? -ne 0 ]
 do
   sleep 5
-  sh $1/wigwag/wwrelay-utils/debug_scripts/create-new-eeprom-with-self-signed-certs.sh\
-    $1/wigwag 8081 $2/userdata/edge_gw_identity
+  exec $1/wigwag/pe-utils/identity-tools/generate-identity.sh\
```

Comment:
```suggestion
  exec "$1/wigwag/pe-utils/identity-tools/generate-identity.sh" \
```
I don't know how easy it is to call this script with a malformed `${1}` parameter, but couldn't multiple commands be injected without encapsulating the command?

Reply:
There is no elevation when running this script, so injecting commands here wouldn't have any benefit.
However, quoting the string will make it work with paths containing spaces, if that is ever the case. Good catch.

Reply:
This release is not moving head of dev forward onto master - it is a subset of dev. Therefore if we want to fix it, we can add the patch on dev and it will get released in the v1.19.1 release or v1.20.0